systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -232,6 +256,24 @@ void testKafkaConnectAndConnectorStateWithFileSinkPlugin() {
     }
 
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Test verifying Kafka Connect functionalities with Plain and SCRAM-SHA authentication."),
+        steps = {
+            @Step(value = "Create object instance of TestStorage", expected = "Instance of TestStorage is created"),
+            @Step(value = "Create NodePools using resourceManager", expected = "NodePools for broker and controller are created"),
```

Comment:
The NodePools are created in each of the testcase (or not, based on configuration). Should this step be anywhere else?

Reply:
True will update this.